main.go

```diff
@@ -153,7 +153,16 @@ func loadPocketArticles(pocketClient *connector.PocketClient, articlesList *[]mo
 }
 
 func printArticle(article models.ArticleResult) {
-	fmt.Printf("-----\n'%s'\n'%s'\n%s\n-----\n", article.ResolvedTitle, article.Excerpt, article.ResolvedURL)
+	minutes := utils.CalculateExpectedReadTime(article.WordCount)
```

Comment:
I could move this out of the print function and instead just pass in the minutes.